components/OrgList.tsx

```diff
@@ -0,0 +1,18 @@
+// TODO: instead of creating another list component, can I use the List.tsx file?
+
+import { Org } from 'interfaces';
+import OrgCard from './OrgCard';
+
+type Props = {
+  items: Org[];
+};
+
+const List: React.FunctionComponent<Props> = ({ items }) => (
+  <>
+    {items.map((item) => (
+      <OrgCard data={item} />
```

Comment:
need to add a `key` prop to items in a list
you can see this error in the browser console:
```
Warning: Each child in a list should have a unique "key" prop.
```